### DISCUSSION TITLE
After migration of server I can not update Transactions

### DISCUSSION THREAD

**Discussion by nomorehugs** [firefly-iii/firefly-iii#6090] [Firefly III]
Hello,
I am not creating a bug because I know it is something in the software, it is something in the server but hosting support can not help me, so I will be glad if you can give some hints.

I am trying to update a transaction but it is not happening, it is giving 403 on the PUT request. When I click on "Store as a new transaction instead of updating. ", the new one is saved withing issue.

Error log from the server:

> [Sun May 22 21:03:35.528125 2022] [authz_core:error] [pid 6750:tid 140534098138880] [client m_IP:36780] AH01630: client denied by server configuration: location_of_the_my_installation/public/api

Debug information generated at 2022-05-22 20:03:57 Europe/Amsterdam for Firefly III version **5.7.6**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.7.6 |
| Firefly III API | 1.5.6 |
| Build | (unknown), (unknown) |
| DB version | 18 (exp. 18) |
| PHP | `8.1.3` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Europe/Amsterdam |
| Browser TZ | Europe/Sofia |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stdout |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | fpm-fcgi |
| Default language | en_US |
| Default locale | equal |
| BCscale | 24 |
| DB drivers | mysql, sqlite, sqlsrv |
| Current driver | mysql |
| Trusted proxies (.env) | `` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start | 2022-05-01 00:00:00 |
| Session end | 2022-05-31 23:59:59 |
| Session first | 2020-01-18 00:00:00 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:100.0) Gecko/20100101 Firefox/100.0 |

**Comment by JC5** [comment]
This looks to be a server configuration issue. 

> hosting support can not help me

I can't tell from what you're sharing why it refuses you access, sorry.
